src/templates/index.html

```diff
@@ -7,6 +7,10 @@
     <link rel="stylesheet" href="static/src/styles/style.css" />
     <link rel="stylesheet" href="static/fonts/Inter/font-inter.css" />
     <title>JoJo</title>
+
+    <script>
+      window.disclaimer= "{{ disclaimer }}";
```

Comment:
It dosent look like any of this code is used

Reply:
Oh, that's... really weird. I meant to use that for parts of the text, but turns out I hardcoded it instead. I'll fix that to use the configured disclaimer instead. Thanks for catching it.